selfdrive/car/toyota/values.py

```diff
@@ -1370,6 +1374,23 @@ class CAR:
     (Ecu.fwdCamera, 0x750, 0x6d): [
       b'8646F7801100\x00\x00\x00\x00',
     ],
+  },
+    CAR.LEXUS_NX_TSS2: {
```

Comment:
fix the indentation

Reply:
The indentation should be corrected now, it had an extra space.